Samples/custom-operator-cpu-sample/customize_model/debug_all_outputs.py

```diff
@@ -0,0 +1,37 @@
+# this script will interlace debug operators for every intermediate output of the onnx graph
+
+import sys
+import os.path as path
+import onnx
+from onnx import helper
+import argparse
+
+def create_modified_model(args):
+    model_path = path.join(path.dirname(path.abspath(__file__)), args.model_path)
+    modified_path = path.join(path.dirname(path.abspath(__file__)), args.modified_path)
+    model = onnx.load(model_path)
+
+    intermediate_outputs = set()
+
+    # gather each unique intermediate output in the graph
+    for node in model.graph.node:
+        for output in node.output:
+            intermediate_outputs.add(output)
+
+    # create a debug operator that consumes each intermediate output
+    # debug operator file path attribute is constructed by the name of the intermediate output
+    for output in intermediate_outputs:
+        inserted_node = model.graph.node.add()
+        valid_output_path = output.replace('/', '\\')

```

Comment:
Just curious if this was necessary? From my experience both '\\\\' and '/' worked to get directory path

Reply:
When I used this script on SqueezeNet which have many outputs which use '/' the file IO failed when running the model with the debug operator provider.